plugins/techdocs-node/src/stages/generate/techdocs.ts

```diff
@@ -143,7 +143,7 @@ export class TechdocsGenerator implements GeneratorBase {
           );
           break;
         case 'docker':
-          await this.containerRunner.runContainer({
+          await this.containerRunner!.runContainer({
```

Comment:
Maybe it's worth throwing an error if the runner is undefined when running in docker?

Reply:
No problem, but are you sure is something that can happen (I mean executing the lines in the if) due to how the classes are created checking the docker option?

Reply:
Ok thanks, I'll fixit tomorrow

Reply:
Done!